Libs/QtTesting/ctkEventTranslatorPlayerWidget.cpp

```diff
@@ -338,10 +359,17 @@ bool ctkEventTranslatorPlayerWidget::compare(const QStringList& actual,
 {
   if (actual != expected)
     {
+    #if (QT_VERSION >= QT_VERSION_CHECK(5,14,0))
+    QTextStream(stderr, QIODevice::WriteOnly)
+        << "Line " << line << " - Problem with function " << function << "\n"
+        << "\tActual value : '" << actualName << "' = " << actual.join(" ") << " \n"
+        << "\tExpected value : '" << expectedName << "' = " << expected.join(" ") << Qt::endl;
```

Comment:
I am wondering if adding `CTK_QT_ENDL` or `CTK_ENDL` to https://github.com/commontk/CTK/blob/master/Libs/Core/ctkCompatibility_p.h would be worth it ?

Reply:
Sure that could be attempted.  This to cut down on the number of duplicated new lines in the diff for this commit?

Reply:
> cut down on the number of duplicated new lines in the diff for this commit

The idea was to keep the content of message consistent between all code paths. 

That said, it is true the change is localized in this file. This could be revisited if this pattern is common.